FrenchFilter/sections/specific.txt

```diff
@@ -2,7 +2,7 @@
 !-------------------- Specific rules --------------------!
 !--------------------------------------------------------!
 !
-rojadirecta.fr,veopartidos.net##.overlay
+veopartidos.net,lowend.xyz###overlay
```

Comment:
`veopartidos.net` is dead on my end. Remove it.

Reply:
@zloyden It's the player in the website.

Reply:
Something is wrong with my VPN. Ok.